.gitmodules

```
@@ -2,3 +2,6 @@
 	path = InvokeAI
 	url = https://github.com/invoke-ai/InvokeAI
 	branch = main
+[submodule "stable-diffusion-webui"]
```

There is an incompatibility between https clone urls and ssh ones - I pulled your branch and was not able to update new submodule. Relative urls make it compatible with both - https://www.damirscorner.com/blog/posts/20210423-ChangingUrlsOfGitSubmodules.html

```suggestion
[submodule "InvokeAI"]
     path = InvokeAI
     url = ../..invoke-ai/InvokeAI
     branch = main
 [submodule "stable-diffusion-webui"]
     path = stable-diffusion-webui
     url = ../../AUTOMATIC1111/stable-diffusion-webui
```